### PR TITLE
Website header footer links

### PR DESCRIPTION
**Pull request by k-sav**
Tackles `Add Spectrum community link on header` & `Add Spectrum and StackOverflow icon links on footer` in #351

**Does this PR introduce a breaking change?**
No

- Added spectrum link to header 
- Added stack overflow logo link to footer 
- Added spectrum logo link to footer 

I wasn't sure about that`MdChat` logo. However, there doesn't appear to be an official logo for spectrum, so I had to improvise. Feel free to change to a more appropriate icon.

### CODE REVIEW

File: packages/website/src/components/Header.tsx [hunk 1]
```diff
@@ -191,6 +191,12 @@ export default function Header({ transparent }: HeaderProps) {
               >
                 Newsletter
               </Anchor>
+              <Anchor
+                href="https://spectrum.chat/reakit"
+                onClick={track("reakit.headerSpectrumClick")}
```
Comment (diegohaz) [inline packages/website/src/components/Header.tsx:197]: To hide it on mobile:
```suggestion
                onClick={track("reakit.headerSpectrumClick")}
                {...props}
```

**Comment by diegohaz** [comment]
Thank you for the PR @k-sav 

I'd like to add the actual Spectrum icon (on header as well, like GitHub), but I didn't find it in SVG.

**Comment by k-sav** [comment]
> Thank you for the PR @k-sav
> 
> I'd like to add the actual Spectrum icon (on header as well, like GitHub), but I didn't find it in SVG.

[Is this ok?](https://gist.github.com/k-sav/0ab65e6b144bf9cf9597e733c09d2f12)

**Comment by diegohaz** [comment]
It's OK. I got this one from Spectrum support (looks the same): https://www.dropbox.com/s/sd30sonb0lfwyir/logo.svg?dl=0

You can put in the icons folder (there are other there).

**Comment by k-sav** [comment]
> It's OK. I got this one from Spectrum support (looks the same): https://www.dropbox.com/s/sd30sonb0lfwyir/logo.svg?dl=0
> 
> You can put in the icons folder (there are other icons there).

Ok @diegohaz I have replaced MdChat with the spectrum logo in the footer and added it to the header, too.

**Pull request merged**